### PR TITLE
fix default Discord username

### PR DESCRIPTION
**Pull request by bbashy**
My Discord notifications stopped working after 9th Jan 2023.

It appears it's because of the default username not being set properly. This change fixes it (username is required, default is Laravel Backup).

Discord returns this error now;
```
"{"username": ["Must be between 1 and 80 in length.", "Username cannot be \"\""]}"
// vendor/spatie/laravel-backup/src/Notifications/Channels/Discord/DiscordChannel.php:16
```

`username` can be set to null or '' so it won't break any existing configs.

### CODE REVIEW

File: config/backup.php [hunk 1]
```diff
@@ -192,14 +192,15 @@
             'webhook_url' => '',
 
             /*
-             * If this is an empty string, the name field on the webhook will be used.
+             * If this is null or an empty string, "Laravel Backup" will be used.
```
Comment (freekmurze) [inline config/backup.php:195]: Let's revert these changes

File: src/Notifications/Channels/Discord/DiscordMessage.php [hunk 2]
```diff
@@ -28,9 +28,11 @@ class DiscordMessage
 
     protected string $url = '';
 
-    public function from(string $username, string $avatarUrl = null): self
+    public function from(string $username = null, string $avatarUrl = null): self
     {
-        $this->username = $username;
+        if (! empty(trim($username))) {
```
Comment (freekmurze) [inline src/Notifications/Channels/Discord/DiscordMessage.php:33]: Let's revert all of the changes in this file. 

Add a check if the username is equal to an empty string, and if so, set the username to Laravel Backup

**Review by freekmurze — CHANGES_REQUESTED** [review]
Let's implement this with less changes.

**Comment by bbashy** [comment]
I'll leave this then. If the username is null or '', it currently doesn't work.

I've set the username in the config which fixed it.
